src/natcap/invest/ui/usage.py

```diff
@@ -11,14 +11,8 @@
 import traceback
 import uuid
 
-try:
-    from urllib.request import urlopen, Request
-    from urllib.parse import urlencode
-    string_types = (str,)
-except ImportError:
-    from urllib2 import urlopen, Request
-    from urllib import urlencode
-    string_types = (str, unicode)
+from urllib.request import urlopen, Request
```

Comment:
Good to get some Python 2.7 stuff out of there!

Reply:
🎉

Reply:
I'm sure there's more to remove, though :)